Models/PFUser+ExtendedUser.m

```diff
@@ -98,12 +98,13 @@ - (void)removeFromWishlist:(GMSPlace*)place {
 - (void)retrieveFavoritesWithCompletion:(void(^)(NSArray<GMSPlace*>*))completion {
     NSMutableArray<GMSPlace*>* places = [NSMutableArray new];
     
-    // if there's no favorites, call completion with nothing
-    if(self.favorites.count <= 0)
+    // if we have no places, return an empty array
+    if ([self.favorites count] == 0)
     {
-        completion(nil);
+        completion([NSArray arrayWithArray:places]);
```

Comment:
[places copy] is even easier - sorry, that's the one I was trying to remember this morning